packages/editor/src/renderer/HTMLRenderer.tsx

```diff
@@ -57,6 +95,10 @@ const HTMLCell: React.FC<
     const cellStyle = getCellStyle(plugin);
     const Provider = plugin.Provider ?? DefaultProvider;
     const data = getCellData(cell, lang);
+    const pluginCellSpacing = getPluginCellSpacing(plugin, data);
+    const [pluginCellSpacingX, pluginCellSpacingY] = pluginCellSpacing
+      ? normalizeCellSpacing(pluginCellSpacing)
```

Comment:
you could `getPluginCellSpacing` alsways normalizing it.

Reply:
@macrozone, getPluginCellSpacing can and should return undefined. I'm checking for it a few lines down. This sequence looks a bit odd but it's due to the fact that hooks can't (or rather shouldn't) be executed conditionally.